### ISSUE TITLE
Allow having multiple ICS files for multi-timeslot events

### ISSUE THREAD

**Issue by oliverklee** [oliverklee/ext-seminars#3037]

**Comment by dreamflasher** [comment]
Either multipe ICS files, or alternatively multiple events in one ICS file – I'd expect the latter one do be easier for users as they only need to import one?

**Comment by oliverklee** [comment]
Yes, if multiple events in one ICS file is possible, we should to that.
